app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -394,9 +397,10 @@ export default {
         }
       } catch (error) {
         if (this.authenticated) {
-          this.isFormScheduleExpired = true;
-          this.isLateSubmissionAllowed = false;
-          this.formScheduleExpireMessage = error.message;
```

Comment:
I was confused about this error handling. It seems like any error in this `getFormSchema()` function would set `this.isFormScheduleExpired` to true, and I am not sure if there is any case where that variable should be set here. I set the error based on the status code, currently only specifying a message for a 401: Unauthorized error. There is also a generic error message for any other cases. I am still testing this to ensure that the `isFormScheduleExpired` variable does not need to be set here.